build/run

```diff
@@ -121,13 +121,15 @@ if [ -z "${DISABLE_NESTED_DOCKER}" ]; then
     -v /var/run/docker.sock:/var/run/docker.sock"
 fi
 
+DOCKER_SERVER_API=$(docker version | grep 'API version' | tail -1 | awk '{print $3}')
```

Comment:
I think using the format field is preferable
```
docker version --format '{{.Client.APIVersion}}'
```

also if the docker daemon is not running for some reason docker version does fail:

```
> docker version --format '{{.Client.APIVersion}}'
1.24
Cannot connect to the Docker daemon. Is the docker daemon running on this host?
```

and given the set -e on this file it would cause the whole run to fail.